Sources/Compiler/TypeChecking/Solution.swift

```diff
@@ -59,16 +59,16 @@ struct Solution {
   var score: Score { Score(errorCount: diagnostics.count, penalties: penalties) }
 
   /// Reifies the given type, substituting each free variable by its corresponding binding.
```

Comment:
```suggestion
  /// Substitutes each free variable for its corresponding binding.
```

Telling me that `reify(x, ...)` “reifies x” is not valuable.
"By" is the wrong word, and "substitutes" is a bit murky in terms of which thing is new/old.  Can we phrase this as "Replaces _oldthing_ with _newthing_?"

Reply:
Yes, we can.